docs/source/testing-and-debugging.md

```diff
@@ -51,9 +51,16 @@ Install [Neo-Express](https://github.com/neo-project/neo-express#installation) a
 
 ### Testing
 
+Before writing your tests, make sure you have a Neo-Express network for local tests.
+Please refer to [Neo-Express documentation](https://github.com/neo-project/neo-express/blob/master/docs/command-reference.md#neoxp-create)
```

Comment:
I kind of feel we could give just enough information that they don't have to first starting reading another link. Something like

"If you do not yet have a local network, open a terminal and type `neoxp create`. Please refer to the Neo-Express documentation for more details on how to configure your local network."

From here they can choose to fine tune their network or just go ahead